src/Sylius/Behat/Service/Accessor/NotificationAccessorInterface.php

```diff
@@ -11,30 +11,20 @@
 
 namespace Sylius\Behat\Service\Accessor;
 
+use Sylius\Behat\Service\NotificationType;
+
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  */
 interface NotificationAccessorInterface
 {
     /**
-     * @return bool
-     */
-    public function hasSuccessMessage();
-
-    /**
-     * @param string $message
-     *
-     * @return bool
-     */
-    public function hasMessage($message);
-
-    /**
      * @return string
      */
     public function getMessage();
 
     /**
-     * @return string
+     * @return NotificationType
      */
-    public function getMessageType();
+    public function getType();
```

Comment:
IMO to be more consistent. `NotificationAccessorInterface::getMessage()` returns the notification message and `NotificationAccessorInterface::getType()` returns the notification type.